bridge/client/sdp.js

```diff
@@ -38,6 +38,7 @@ if (typeof(SDP) == "undefined")
         "tline": "^t=([\\d]+) ([\\d]+).*$",
         "cline": "^c=IN (IP[46]) ([\\d\\.a-f\\:]+).*$",
         "msidsemantic": "^a=msid-semantic: *WMS .*$",
+        "iceoptions": "^a=ice-options:trickle",
```

Comment:
We should probably discard any junk at the end like the other patterns do. I.e. add .*$ at the end.